src/scripts/webpack/build-ts.js

```diff
@@ -0,0 +1,14 @@
+import webpackBuild from './webpack.build.js';
+import compiler from './compiler';
+import { logger } from '../../util';
+import { statsOutputConfiguration } from './util';
+import replaceBabelWithTypescript from './replaceBabelWithTypescript';
+
+const getBuildConfig = (...args) => replaceBabelWithTypescript(
```

Comment:
can't we make `webpack.base.js` parametric instead? Aka take a param `compiler=TS | Babel`, and build `loaders` based on it?